Oryx.Cognite/src/Handler.fs

```diff
@@ -181,7 +181,15 @@ module HttpHandler =
                         error.RequestId <- requestId
                     | None -> ()
 
-                    return error.ToException()
+                    let ex: exn =
```

Comment:
Should we instead make some optional arguments for `ToException` so you can set the message and code if needed?

Reply:
Probably better, yeah. I'll make the change.